xwiki-platform-core/xwiki-platform-model/xwiki-platform-model-validation/xwiki-platform-model-validation-api/pom.xml

```diff
@@ -30,7 +30,7 @@
   <description>XWiki Platform - Model Validation API to handle model validation in XWiki</description>
   <packaging>jar</packaging>
   <properties>
-    <xwiki.jacoco.instructionRatio>1.00</xwiki.jacoco.instructionRatio>
+    <xwiki.jacoco.instructionRatio>0.97</xwiki.jacoco.instructionRatio>
```

Comment:
Decreased due to the introduction of `cleanConfigurationCache`, a default method with an empty body.

Reply:
Note that default methods should ideally be tested too ;)

Reply:
I agree but I'm not sure how to test a method with an empty body :/

Reply:
Indeed :) I've replied at https://github.com/xwiki/xwiki-platform/pull/1636/files#r635320431

Reply:
> but I'm not sure how to test a method with an empty body :/

Not a very useful test, but I guess all it takes in terms of coverage is to call it in some test.